app/geo/gui/src/ts/twxplore/gui/geo/states/map/FeatureAttributeName.ts

```diff
@@ -0,0 +1,14 @@
+export enum FeatureAttributeName {
+    label = "label",
```

Comment:
Nit: by convention enumerators are CAPITALIZED_NAMES, but don't worry about it in this case.

Reply:
I had them capital at first. The problem was that the attribute string that out MapFeature objects have is lowercase. So if I use that string as a key for the enum it returns undefined. I guess I could use something like .UpperCase() on the strings, but that seemed very messy.

Reply:
Per my other comment, your valueOf should take a string or the enum.
You should handle the string -> enum conversion in one place rather than duplicating that code.